WickedEngine/wiScene_BindLua.cpp

```diff
@@ -4,6 +4,13 @@
 #include "wiEmittedParticle.h"
 #include "wiTexture_BindLua.h"
 #include "wiPrimitive_BindLua.h"
+#include <LUA/lua.h>
+#include <Utility/DirectXMath.h>
```

Comment:
You must not include this, it is causing compile error on windows. On windows, we use a different version of math library. You can include wiMath.h to use math library.